config/app/app.yaml

```diff
@@ -16,3 +16,6 @@ parameters:
     ngsite.default.token_validity.email_change: 900
 
     app.testing.site_domain: '%env(TEST_DOMAIN)%'
+
+    fos_ckeditor.base_path: 'bundles/netgensurveybuilder/ckeditor4'
```

Comment:
Shouldn't these be part of the survey bundle? They don't change ever, after all?

@hknezevic ?